ldj-spring-renderer-example/src/test/java/com/infosupport/ldoc/springexample/SpringEventRendererTest.java

```diff
@@ -0,0 +1,139 @@
+package com.infosupport.ldoc.springexample;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.infosupport.ldoc.springexample.SpringEventRenderer.Interaction;
+import com.infosupport.ldoc.springexample.SpringEventRenderer.Kind;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class SpringEventRendererTest {
+
+  private final StringWriter sw = new StringWriter();
+  private final PrintWriter pw = new PrintWriter(sw);
+
+  @Test
+  void constructor() {
+    assertDoesNotThrow(SpringEventRenderer::new);
```

Comment:
Why this test?

Reply:
Ah, I had it to check that JUnit was working, it doesn't serve a purpose anymore. Removed.